pkgs/by-name/pl/plutosvg/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+  validatePkgConfig,
+  testers,
+  cmake,
+  ninja,
+  plutovg,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "plutosvg";
+  version = "0.0.7";
+
+  src = fetchFromGitHub {
+    owner = "sammycage";
+    repo = "plutosvg";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-4JLk4+O9Tf8CGxMP0aDN70ak/8teZH3GWBWlrIkPQm4=";
+  };
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  patches = [
+    # https://github.com/sammycage/plutosvg/pull/29
+    ./0001-Emit-correct-pkg-config-file-if-paths-are-absolute.patch
```

Comment:
Sorry for not seeing this before, but the includedir is mentioning the `.out` path, when it should be the `.dev` path.

Reply:
No need @marcin-serwin, I've added a postInstall fix. Good work!